From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: iucv: use correct output register in iucv_query_maxconn()
References: bnc#533267,LTC#56116

Symptom:     The system log contains kernel messages reporting that the
             iucv "pathid" is greater than "max_connections".
Problem:     When querying the maximum number of iucv connections, the
             wrong output register is used.
Solution:    Use the correct output register when querying the maximum
	     number of IUCV connections.

Acked-by: John Jolly <jjolly@suse.de>
---
 net/iucv/iucv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/net/iucv/iucv.c
+++ b/net/iucv/iucv.c
@@ -334,7 +334,7 @@ static int iucv_query_maxconn(void)
 		"	srl	%0,28\n"
 		: "=d" (ccode), "+d" (reg0), "+d" (reg1) : : "cc");
 	if (ccode == 0)
-		iucv_max_pathid = reg0;
+		iucv_max_pathid = reg1;
 	kfree(param);
 	return ccode ? -EPERM : 0;
 }
